### PR TITLE
[locale] tg-TJ: Improve months and date formats

### PR DESCRIPTION
**Pull request by orif-jr**
- extend months with standalone and formatted options
- fix long date formats according to local standards
- update the word "tomorrow" with widely used synonym

### CODE REVIEW

File: src/test/locale/tg.js [hunk 1]
```diff
@@ -16,14 +17,6 @@ test('parse', function (assert) {
         );
     }
 
-    function equalTestStrict(input, mmm, monthIndex) {
```
Comment (marwahaha) [inline src/test/locale/tg.js:19]: Why are these tests removed?

Comment (marwahaha) [reply src/test/locale/tg.js:19]: @orif-jr

Comment (orif-jr) [reply src/test/locale/tg.js]: This code block was newer used for tests. So, it's [blocked the builds always](https://travis-ci.org/github/moment/moment/jobs/727752429#L234). I reverted the code block back and added some more tests.

**Comment by marwahaha** [comment]
Great!

**Pull request merged**